packages/mimir/src/rules/no-useless-spread.ts

```diff
@@ -0,0 +1,24 @@
+import * as ts from 'typescript';
+import { AbstractRule, Replacement } from '@fimbul/ymir';
+
+const FAILURE_STRING = 'Using the spread operator here is not necessary.';
+
+export class Rule extends AbstractRule {
```

Comment:
If you add the `@excludeDeclarationFiles` decorator this rule will automatically skip all declaration files. Since spread is not possible in declaration files, this saves a few CPU cycles.